scipy/optimize/_linprog_highs.py

```diff
@@ -195,6 +195,25 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
                 The number of primal/dual pushes performed during the
                 crossover routine for ``solver='ipm'``.  This is ``0``
                 for ``solver='simplex'``.
+            marginals : OptimizeResult
```

Comment:
This needs to be moved/copied over to `_linprog_doc.py` to appear in the documentation.

Reply:
Copied over to `_linprog_doc.py`

Reply:
This went into the "Options" of  `method='highs'` only. It should be in"Returns" of  `method='highs'`, `method='highs-ipm'`, and `method='simplex'`.

Reply:
Yep, I did this incorrectly, thanks for seeing that.  I'll have time to fix this and review your comment below in the next few days

Reply:
Latest push moves the entry from Options -> Results in `method='highs'` and adds the same entry to `method='highs-ipm` and `method='highs-ds`